perfkitbenchmarker/vm_util.py

```diff
@@ -194,243 +188,9 @@ def GetSshOptions(ssh_key_filename):
-    raise ValueError('If Param is a tuple, the tuple must be (tuple, dict)')
-  else:
-    target_arg_tuples = [(target, args, kwargs)
-                         for args, kwargs in thread_params]
-
-  return RunParallelThreads(target_arg_tuples,
-                            max_concurrency=max_concurrent_threads)
-
-
-def _ExecuteProcCall(target_arg_tuple):
-  """Function invoked in another process by RunParallelProcesses.
-
-  Executes a specified function call and captures the traceback upon exception.
-  TODO(skschneider): Remove this helper function when moving to Python 3.5 or
-  when the backport of concurrent.futures.ProcessPoolExecutor is able to
-  preserve original traceback.
-
-  Args:
-    target_arg_tuple: (target, args, kwargs) tuple containing the function to
-        call and the arguments to pass it.
-
-  Returns:
-    (result, traceback) tuple. The first element is the return value from the
-    called function, or None if the function raised an exception. The second
-    element is the exception traceback string, or None if the function
-    succeeded.
-  """
-  target, args, kwargs = target_arg_tuple
-  try:
-    return target(*args, **kwargs), None
-  except:
-    return None, traceback.format_exc()
-
-
-def RunParallelProcesses(target_arg_tuples, max_concurrency=None):
-  """Executes function calls concurrently in separate processes.
-
-  Args:
-    target_arg_tuples: list of (target, args, kwargs) tuples. Each tuple
-        contains the function to call and the arguments to pass it.
-    max_concurrency: int or None. The maximum number of concurrent new
-        processes. If None, it will default to the number of processors on the
-        machine.
-
-  Returns:
-    list of function return values in the order corresponding to the order of
-    target_arg_tuples.
-
-  Raises:
-    errors.VmUtil.CalledProcessException: When an exception occurred in any
-        of the called functions.
-  """
-  call_futures = []
-  results = []
-  error_strings = []
-  with futures.ProcessPoolExecutor(max_workers=max_concurrency) as executor:
-    for target_arg_tuple in target_arg_tuples:
-      call_futures.append(executor.submit(_ExecuteProcCall, target_arg_tuple))
-    for index, future in enumerate(call_futures):
-      try:
-        result, stacktrace = future.result()
-      except:
-        result = None
-        stacktrace = traceback.format_exc()
-      results.append(result)
-      if stacktrace:
-        msg = ('Exception occurred while calling {0}:{1}{2}'.format(
-            _GetCallString(target_arg_tuples[index]), os.linesep, stacktrace))
-        logging.error(msg)
-        error_strings.append(msg)
-  if error_strings:
-    msg = ('The following exceptions occurred during parallel execution:'
-           '{0}{1}'.format(os.linesep, os.linesep.join(error_strings)))
-    raise errors.VmUtil.CalledProcessException(msg)
-  return results
+RunParallelProcesses = background_tasks.RunParallelProcesses
```

Comment:
Doesn't look like there are any references to vm_util.RunParallelProcesses (or background_tasks.RunParallelProcesses for that matter). Who/what do we expect to use this?